app/assets/javascripts/full_calendar.js

```diff
@@ -3,13 +3,13 @@ window.onload = function () {
        calendar()
     }
     else {
-        metricsGraph('client_strength');
-        metricsGraph('client_anthropometry');
+        $('canvas[data-chart]').each(function (index, canvas) {
```

Comment:
read about jquery and stop doing this shit
http://learn.jquery.com/using-jquery-core/document-ready/